contracts/options.cairo

```diff
@@ -165,6 +166,19 @@ func mint_option_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_che
                 underlying_price=underlying_price,
             );
         }
+
+        with_attr error_message("trade exceeds max liqpool deposit size"){
+            let (user_address) = get_caller_address();
+            let (currency_address) = get_underlying_token_address(lptoken_address);
+            let (optsize) = IOptionToken.balanceOf(contract_address=option_token_address, account=user_address);
+            assert optsize.high = 0;
+            let (adjspd) = get_pool_volatility_adjustment_speed(lptoken_address=lptoken_address);
+            let adjspd_ = adjspd * 100;
+            assert_nn(adjspd_);
+            let max_optsize_m64x61 = adjspd_ /  MAX_OPTION_SIZE_PERCENT_OF_VOLADJSPD;
```

Comment:
shouldn't it be `*` instead of `/` since the MAX_OPTION_SIZE_PERCENT_OF_VOLADJSPD in %? or even better

`adjspd_ *  MAX_OPTION_SIZE_PERCENT_OF_VOLADJSPD / 100`